lib/orangetheses/indexer.rb

```diff
@@ -91,6 +91,14 @@ def title(dc_elements)
       titles.empty? ? nil : titles.first.text
     end
 
+    def title_sort(dc_elements)
+      titles = dc_elements.select { |e| e.name == 'title' }
+      title = titles.empty? ? nil : titles.first.text
+      unless title.nil?
+        title.downcase.gsub(/\p{^Alnum}/, '').gsub(/^(?:a)|(?:an)|(?:the)/, '')
```

Comment:
I think we want to change this regex to: `title.downcase.gsub(/^(a|an|the)\s/, '')` for better readability and correct output. We don't need to handle non-alphanumerics here because they get stripped in [Solr](https://github.com/pulibrary/orangelight/blob/development/solr_conf/conf/schema.xml#L481). I think we also need to keep the spaces in the regex to prevent words like "thesis" to get stripped down to "sis."
```ruby
title = "Thesis: Anyone out there reading this?"
title.downcase.gsub(/\p{^Alnum}/, '').gsub(/^(?:a)|(?:an)|(?:the)/, '')
 => "sisyoneoutrereadingthis"
title.downcase.gsub(/^(a|an|the)\s/, '')
 => "thesis: anyone out there reading this?"

# (stripping alphanumeric for better comparison)
title.downcase.gsub(/^(a|an|the)\s/, '').gsub(/\p{^Alnum}/, '')
 => "thesisanyoneouttherereadingthis" 


title = "An awesome thesis"
title.downcase.gsub(/^(a|an|the)\s/, '')
 => "awesome thesis" 
```